oioioi/teachers/tests.py

```diff
@@ -181,3 +185,36 @@ def test_teacher_modify(self):
         self.assertEqual(Teacher.objects.all().count(), 2)
         mod_teacher = Teacher.objects.get(pk=1001)
         self.assertEqual(mod_teacher.school, "New School")
+
+
+class TestAddUserToContestForm(TestCase):
```

Comment:
I would also prefer to have tests that use HTTP requests as well including test that double checks if it is not possible to add users/teachers as regular user